src/mame/namco/namcos10.cpp

```diff
@@ -3026,6 +3057,28 @@ static INPUT_PORTS_START( squizchs )
 
 INPUT_PORTS_END
 
+static INPUT_PORTS_START( ptblank3 )
+	PORT_INCLUDE(namcos10)
+
+	PORT_MODIFY("IN1")
+	PORT_BIT( 0x0fff6f6f, IP_ACTIVE_LOW, IPT_UNUSED )
+	PORT_BIT( 0x00000010, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(1) PORT_NAME("P1 Trigger")
+	PORT_BIT( 0x00001000, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(2) PORT_NAME("P2 Trigger")
```

Comment:
Do you really need to give these names? They won’t get localised names by default that way.

Reply:
I guess since it's just a single button it's not a big deal that it's not labeled individually.